docs/reference/release-notes/highlights.asciidoc

```diff
@@ -25,19 +25,20 @@ Other versions:
 | {ref-bare}/7.1/release-highlights-7.1.0.html[7.1]
 | {ref-bare}/7.0/release-highlights-7.0.0.html[7.0]
 
+// tag::notable-highlights[]
+[discrete]
+[[eql-like-regex-operators]]
+=== EQL: `like` and `regex` operators
 
+In 7.12, we added the `like` and `regex` operators to EQL. You can use the
+`like` operator to match strings to
+{ref}/eql-syntax.html#eql-syntax-wildcards[wildcard patterns], such as `foo*` or
+`ba?`. You can use the `regex` operator to match strings to regular expressions.
+You can use both `like` and `regex` as lookup operators to match a string
+against a list of patterns.
 
-// Use the notable-highlights tag to mark entries that
-// should be featured in the Stack Installation and Upgrade Guide:
-// tag::notable-highlights[]
-// [discrete]
-// === Heading
-//
-// Description.
+By default, both operators are case-sensitive. For case-insensitive matching,
+use `like~` or `regex~`. For more information, see the
+{ref}/eql-syntax.html#eql-syntax-pattern-comparison-operators[Pattern comparison
+operators] section of the {ref}/eql-syntax.html[EQL syntax documentation].
```

Comment:
I opened this PR against 7.x to include this release highlight. I'll remove this in the 8.0 backport.

Reply:
> `like` and `regex` operators
`like` and `regex` are keywords for lack of a better word not operators.